src/content/eth2/why-eth2/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Why Eth2?
+description: Everything you need to know about why Ethereum needs Eth2
+lang: en
+sidebar: true
+---
+
+# Why Eth2?
+
+<Subtitle>In the five years since launch, Ethereum has shown us what’s possible when you combine blockchain technology and software applications to create open, permissionless systems. But Ethereum still needs to scale to fulfil its potential.</Subtitle>
+
+Today, Ethereum handles a tremendous amount of value and it works well. But the design of Eth2 was always the vision – it just wasn't possible at the time Ethereum launched. Eth2 will solve the compromises of [Ethereum 1.0](/what-is-ethereum/).
+
+## Current limitations
+
+### Speed ⚡️
+
+Ethereum can only handle 7 - 15 transactions per second. For comparison Visa processes around 1500. Right now, Ethereum couldn’t handle mainstream adoption.
+
+### Energy 💡
+
+Verifying transactions is currently very energy intensive which isn’t sustainable.
+
+### Opportunity 🏗
+
+The upfront costs of mining equipment means there’s a a substantial financial barrier to participating in the network.
+
+Some blockchains have claimed to solve some or all of these issues. But this is often at the expense of principles that Ethereum won't ever compromise on, like decentralization.
+
+## The Ethereum trilemma
+
+The Ethereum trilemma is that it's inherently difficult to scale a blockchain in a secure, decentralized way.
+
+To make ETH more scalable it will have to be faster and less energy-intensive. This wouldn't be as tricky if you were willing to compromise on decentralization. But when you compromise decentralization you also start to compromise the openness and security of the system. This is a big no, no for Ethereum.
```

Comment:
```suggestion
To make ETH more scalable it will have to be faster and less energy-intensive. This wouldn't be as tricky if you were willing to compromise on decentralization. But when you compromise decentralization you also start to compromise the openness and security of the system. This is a big no-no for Ethereum.
```